packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/data_sources/local/open_mail_app_local_data_source.dart

```diff
@@ -0,0 +1,19 @@
+{{> licence.dart }}
+
+import 'package:open_mail/open_mail.dart';
+
+import '../../models/errors/error_model.dart';
+
+class OpenMailAppDataSource {
```

Comment:
Would be great if we had this documented.

Reply:
Done